webapp-django/crashstats/crashstats/models.py

```diff
@@ -447,6 +450,9 @@ def _post(self, url, payload, method="post"):
         # set dont_cache=True here because the request depends on the payload
         return self.fetch(url, method=method, data=payload, dont_cache=True)
 
+    def _options(self, method="options"):
+        return {}, False
```

Comment:
I don't understand the second bit here. This mimics what `fetch` returns and there's no comments around why that returns a dict and a bool. I need to follow up on this. But I'll do it another day.